docs/source/use_with_pytorch.mdx

```diff
@@ -86,20 +86,36 @@ To get a single tensor, you must explicitly use the [`Array`] feature type and s
 {'data': tensor([0, 0, 1])}
 ```
 
-However, since it's not possible to convert text data to PyTorch tensors, you can't format a `string` column to PyTorch.
-Instead, you can explicitly format certain columns and leave the other columns unformatted:
+String and binary objects are unchanged, since PyTorch only supports numbers.
+
+The [`Image`] and [`Audio`] feature types are also supported:
```

Comment:
I would also add an example with a slice to show that batches (of images) can also be formatted.